src/domain/occurrence/CreateOccurrencePage.tsx

```diff
@@ -179,4 +229,33 @@ const CreateOccurrencePage: React.FC = () => {
   );
 };
 
+const useInitialFormValues = () => {
```

Comment:
Is this same hook also needed on edit page? If yes, you could move this to separate file

Reply:
at least not yet, I will move it if it is needed. But for now only create occurrence page needs initialValues from query params

Reply:
Ok, in that case there is no need to do anything for this